src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -192,21 +194,35 @@ class DeepDiveLogParser extends JavaTokenParsers {
     | lexpr
     )
 
-  def lexpr : Parser[Expr] =
-    ( functionName ~ "(" ~ rep1sep(expr, ",") ~ ")" ^^ {
-        case (name ~ _ ~ args ~ _) => FuncExpr(name, args, (aggregationFunctions contains name))
+  def cexpr =
+    ( expr ~ compareOperator ~ expr ^^ { case (lhs ~ op ~ rhs) => BinaryOpExpr(lhs, op, rhs) }
+    | expr
+    )
+
+  def lexpr =
+    ( "if" ~> (cond ~ ("then" ~> expr) ~ rep(elseIfExprs) ~ opt("else" ~> expr)) <~ "end" ^^ {
```

Comment:
Keywords use upper case letters elsewhere. I think it's better to unify them (upper/lower/ignore case).